src/api/utils/fs_utils.ts

```diff
@@ -31,16 +31,24 @@ export function rimraf(thing: string) {
 }
 
 export function copy(from: string, to: string) {
-	if (!fs.existsSync(from)) return;
-
-	const stats = fs.statSync(from);
-
-	if (stats.isDirectory()) {
+	const copied = new Set();
+	const startDir = from + "/";
+	copyFiles(from, to);
```

Comment:
It's bad practice to call a function before it's declared. It also might make sense to not create a new method for this behavior, since the signature is still `from` and `to`. I would imagine copy would just call itself.

Reply:
It's just common design practice to declare functions before you call them. Similar to how Imports go at the top of the page. Yes, JS can hoist for you. But in general, like all programming languages, it's better to organize it different.